python/lib/sift_grafana/sift_query_model.py

```diff
@@ -0,0 +1,56 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, Field, model_validator
```

Comment:
This will also need to be added to `pyproject.toml` [here](https://github.com/sift-stack/sift/blob/735ddca3ea26e22142c29518b5c32d819b968c5a/python/pyproject.toml#L23).

You can add `pydantic~=2.0.0` which is short hand for anything between 2.0.0 and 3.0.0.

If `2.0.0` breaks for whatever reason try `2.1.0`, `2.2.0`, and so on.